crates/frontend/src/api.rs

```diff
@@ -194,3 +194,16 @@ pub async fn fetch_experiment(
         Err(e) => Err(ServerFnError::ServerError(e.to_string())),
     }
 }
+
+pub async fn delete_default_config(key: String, tenant: String) -> Result<(), String> {
+    let host = get_host();
+    let url = format!("{host}/default-config/{key}");
+
+    request(
```

Comment:
@ShubhranshuSanjeev You would need to fix the request function a bit as it tries to return response message, and as the delete function as nothing in response it triggers a snack bar with the following error.
<img width="1728" alt="Screenshot 2024-05-17 at 11 55 29 AM" src="https://github.com/juspay/superposition/assets/79959102/0cb39b8b-6655-4311-b183-1233ea3a9e00">

Reply:
Sure, let me take a look, and thanks for pointing this, I missed it.